introduction.md

```diff
@@ -60,10 +60,6 @@ You can watch a YouTube playlist of getting started tutorials for Blazor (videos
 
 Use [our feedback portal](https://feedback.telerik.com/blazor) to tell us which components you want us to create and how you intend to use them with Blazor.
 
-## Building Blocks and Page Templates
```

Comment:
You can wrap the content in a plain HTML comment, which will make restoration easier. `<!-- ... -->`

Reply:
I saved it locally in another way.